src/GifPickerReact.css

```diff
@@ -28,7 +27,7 @@
     --gpr-category-list-padding: var(--gpr-horizontal-padding);
     --gpr-category-border-radius: 4px;
     --gpr-category-background-opacity: 40%;
-    --gpr-category-font-color: var(--gpr-picker-border-color);
+    --gpr-category-font-color: #e7e7e7;
```

Comment:
Still useless change, just rever it

Reply:
This is needed for DarkMode otherwise the categories are hard to read because it was relying on border color before.

Reply:
removed:
![image](https://user-images.githubusercontent.com/107433287/212143808-5059ca00-8aa8-4a67-b9dc-e6133e7b53b7.png)

changed:
![image](https://user-images.githubusercontent.com/107433287/212143899-16fdf509-8ee3-46d8-887b-c06e2129567c.png)